### ISSUE TITLE
Can't search forked repos

### ISSUE THREAD

**Issue by jderose9** [microsoft/Code-Search#81]
I can't enable searching for a forked repository. We use forks in our organization to allow different business units to build features and be merged into the parent repo once approved. We need to be able to search the forks as well.

**Comment by ywkao** [comment]
I totally agree with @jderose9. In my case, several branches are developed in a forked repository, and it is really time-consuming when looking for keywords through files. I think it will be much more user-friendly if this function can be enabled.

**Comment by sbingham-MET** [comment]
This issue has become super important for our org. Not sure why it has not been addressed yet. Anyone from MSFT able to help escalate this issue?

There is a feature request people can upvote here: https://developercommunity.visualstudio.com/t/enable-searching-in-git-forks-the-same-way-as-git/366522

**Comment by CLazar-MET** [comment]
This someone doubles as an issue in Azure DevOps (or any similar platform that utilizes Git Wikis). All of the wikis created from forked repos will also not be searchable. In the case of Azure DevOps, this could be publishing code repositories as wikis for documentation purposes.
